UPGRADE-4.4.md

```diff
@@ -76,6 +76,11 @@ MonologBridge
 
  * The `RouteProcessor` has been marked final.
 
+PropertyAccess
+--------------
+
+ * Deprecated to pass `null` as `$defaultLifetime` 2nd argument of `PropertyAccessor::createCache()` method, pass 0 instead.
```

Comment:
`passing`?

Reply:
Hmm, maybe... @ocrampete16 Can you help out?

Reply:
"Deprecated passing `null`" is correct. 👍

```suggestion
 * Deprecated passing `null` as 2nd argument of `PropertyAccessor::createCache()` method (`$defaultLifetime`), pass 0 instead.
```

Reply:
Thanks Marco :-)